packages/bits/demo/src/components/demo/wizard-v2/wizard-docs/wizard-docs.component.html

```diff
@@ -104,6 +104,17 @@ <h3>Wizard Step Change Event Example</h3>
     <nui-wizard-step-change-example></nui-wizard-step-change-example>
 </nui-example-wrapper>
 
+<h3>Wizard Tooltip</h3>
+<p>
+    You are able add tooltip to wizard. By default if label text overflow container we show tooltip.
+    Note that it doesn't work with label with custom template. In this case you should add tooltip explicitly. For example:
+    <br>
+</p>
+<pre>< nui-wizard-step-v2 tooltip="Custom Label Template" ></pre>
```

Comment:
I feel like we can leave out this code snippet, and just point consumers to the source code in the example wrapper like we usually do.